### PR TITLE
Add GitHub Actions Workflows To Manage Issues

### PR DESCRIPTION
**Pull request by ihcsim**
## Change Overview

This PR introduces 2 GitHub Actions workflows to manage new and stale issues. 

The `stale` workflow is scheduled to run nightly to identify issues that are older than 60 days as `stale`. Stale issues that have been inactive for more than 30 days are automatically closed. Issues that are labelled as `frozen` are exempted from this clean-up cycle.

The `triage` workflow automatically adds all new issues and pull requests to the Kanister project. New issues will be assigned to the "To Be Triaged" column, while pull requests go under the "In Progress" column. It also labels the issue as `triage` and adds an acknowledgement comment to the issues and pull requests. 

This is what a new issue looked like after the `triage` workflow has processed it:

image

The issue has been automatically added to the project board:

image

### CODE REVIEW

File: .github/workflows/triage.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  pull_request:
+    types:
+    - opened
+    - reopened
+jobs:
+  issues:
+    if: github.event_name == 'issues'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: andymckay/labeler@1.0.2
```
Comment (pavannd1) [inline .github/workflows/triage.yaml:15]: Looks good 👍🏼  Are these actions/images safe and widely used?

Comment (ihcsim) [reply .github/workflows/triage.yaml:15]: Yeah, I found them in the GHA docs at: 
* [adding labels to issues](https://docs.github.com/en/actions/managing-issues-and-pull-requests/adding-labels-to-issues)
* [moving assigned issues to project](https://docs.github.com/en/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards)

The labeler action is used by more than 6.3k pipelines, per their GH repo stats.

Comment (julio-lopez) [reply .github/workflows/triage.yaml:15]: same question

File: .github/workflows/triage.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  issues:
+    types:
+    - opened
+    - reopened
+  pull_request:
+    types:
+    - opened
+    - reopened
+jobs:
+  issues:
```
Comment (julio-lopez) [inline .github/workflows/triage.yaml:11]: Should this be added to reduce the permissions scope of the token?

```suggestion
  issues:
    permissions:
      issues: write
```

File: .github/workflows/triage.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+          If this is a bug report, make sure to include clear instructions how on to reproduce the problem with [minimal reproducible examples](https://stackoverflow.com/help/minimal-reproducible-example), where possible. If this is a security report, please review our security policy as outlined in [SECURITY.md](https://github.com/kanisterio/kanister/blob/master/SECURITY.md).
+
+          If you haven't already, please take a moment to review our project's [Code of Conduct](https://github.com/kanisterio/kanister/blob/master/CODE_OF_CONDUCT.md) document.
+    - uses: alex-page/github-project-automation-plus@v0.8.1
+      with:
+        repo-token: ${{ secrets.GH_TOKEN }} # must use a PAT here
+        project: Kanister
+        column: To Be Triaged
+  pull_requests:
```
Comment (julio-lopez) [inline .github/workflows/triage.yaml:35]: Similar to the comment above

```suggestion
  pull_requests:
    permissions:
      pull-requests: write
```

File: .github/workflows/triage.yaml (hunk 1 above)
Comment (ihcsim) [reply .github/workflows/triage.yaml:15]: These are the ones used in the GHA docs. Most of the other actions found in the marketplace are not endorsed by GitHub. There are some more "official looking" ones, with tests:

* https://github.com/actions-ecosystem/action-add-labels
* https://github.com/actions-ecosystem/action-create-comment
* https://github.com/actions/add-to-project (I think this works only with Projects Beta)

Other than that, they are just as old, with similar number of issues, forks, stars etc.

Comment (ihcsim) [reply .github/workflows/triage.yaml:15]: Do we have any additional selection criteria?

File: .github/workflows/triage.yaml (hunk 2 above)
Comment (ihcsim) [reply .github/workflows/triage.yaml:11]: Good point; will fix :+1:.

File: .github/workflows/triage.yaml (hunk 1 above)
Comment (ihcsim) [reply .github/workflows/triage.yaml]: I changed the 'labelling' and 'commenting' action plugins to the ones from `actions-ecosystem`. If anything, they at least have tests to cover their code.

**Review by pavannd1 — APPROVED** [review]
LGTM

**Pull request merged**